docs/Deploying/Configuration.md

```diff
@@ -3057,7 +3057,7 @@ The new account's email and full name are copied from the user data in the SSO r
 By default, accounts created via JIT provisioning are assigned the [Global Observer role](https://fleetdm.com/docs/using-fleet/permissions).
 To assign different roles for accounts created via JIT provisioning see [Customization of user roles](#customization-of-user-roles) below.
 
-To enable this option, go to **Settings > Organization settings > single sign-on options** and check "_Automatically create Observer user on login_" or [adjust your config](#sso-settings-enable-jit-provisioning).
+To enable this option, go to **Settings > Organization settings > single sign-on options** and check "Create user and sync permissions on login_" or [adjust your config](#sso-settings-enable-jit-provisioning).
```

Comment:
Small typo `.... on login_`

Reply:
Sorry I don't see the typo, is it the `_`? (I probably missed the starting `_` that turns this into _cursive_)